dspace-api/src/main/java/org/dspace/browse/BrowserScope.java

```diff
@@ -354,7 +360,7 @@ public int getResultsPerPage()
      */
     public void setResultsPerPage(int resultsPerPage)
     {
-        if (resultsPerPage > -1 || (browseIndex != null && browseIndex.isTagCloudEnabled()))
```

Comment:
This change is essentially a revert of the previous commit. Is it intentional or were you accidentally working with older source?